python/ray/dashboard/node_stats.py

```diff
@@ -16,6 +16,59 @@
 
 logger = logging.getLogger(__name__)
 
+PYCLASSNAME_RE = re.compile(r"(.+?)\(")
+
+
+def group_actors_by_python_class(actors):
+    groups = defaultdict(list)
+    for actor in actors.values():
+        actor_title = actor.get("actorTitle")
+        if not actor_title:
+            groups["Unknown Class"].append(actor)
+        else:
+            match = PYCLASSNAME_RE.search(actor_title)
+            if match:
+                # Catches case of actorTitle like
+                # Foo(bar, baz, [1,2,3]) -> Foo
+                class_name = match.groups()[0]
+                groups[class_name].append(actor)
+            else:
+                # Catches case of e.g. just Foo
+                # in case of actor task
+                groups[actor_title].append(actor)
+    return groups
+
+
+def get_actor_group_stats(group):
+    state_to_count = defaultdict(lambda: 0)
+    executed_tasks = 0
+    min_timestamp = None
+    num_timestamps = 0
+    sum_timestamps = 0
+    now = time.time() * 1000  # convert S -> MS
+    for actor in group:
+        logger.info(actor)
+        state_to_count[actor["state"]] += 1
+        if "timestamp" in actor:
+            if not min_timestamp or actor["timestamp"] < min_timestamp:
+                min_timestamp = actor["timestamp"]
+            num_timestamps += 1
+            sum_timestamps += now - actor["timestamp"]
+        if "numExecutedTasks" in actor:
```

Comment:
Any reason why we are only collecting `NumExecutedTasks` but not others?

Reply:
`numPendingTasks` ended up, as far as I can tell, always being 0. 
There are some other metrics we'd like to later display, but most of them are either nontrivial to implement or are lower priority than other things.
https://docs.google.com/document/d/14c4Sw8G9wc_diXBnyUz1tPFh896iuWLr9saBW0Gl-Jc/edit#heading=h.u4av5dyeaceu